Data/ByteString/Unsafe.hs

```diff
@@ -25,7 +25,9 @@ module Data.ByteString.Unsafe (
         unsafeLast,
         unsafeIndex,
         unsafeTake,
+        unsafeTakeEnd,
         unsafeDrop,
+        unsafeDropEnd,
```

Comment:
This module is publicly exposed, so please update changelog and provide `@since` annotations.